packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -39,6 +38,16 @@ const String kBaseHref = 'baseHref';
 /// The caching strategy to use for service worker generation.
 const String kServiceWorkerStrategy = 'ServiceWorkerStrategy';
 
+List<String> updateDartDefines(List<String> dartDefines, WebRendererMode webRenderer) {
```

Comment:
nit: Is this exposed for use anywhere outside of this file other than for testing? If so, shouldn't this be marked with @visibleForTesting (or perhaps more ideally, covered indirectly by tests on `Dart2JSTarget` and `Dart2WasmTarget`)?

Reply:
At this point this is both directly (`web_defines_test.dart`) and indirectly (`web_test.dart`) tested by unit tests. I'll add the `@visibleForTesting` annotation.